package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -330,6 +330,9 @@ def _get_dt(self):
             t1 = self.ts.time
             dt = t1 - t0
         except StopIteration:
+            msg = ('dt information could not be obtained, defaulting to 0 ps. '
+                   'Note: in MDAnalysis 2.1.0 this default will change 1 ps.')
+            warnings.warn(msg)
             return 0
```

Comment:
This is inconsistent with what we do in base.py, ideally this should just throw an AttributeError and default on what's done in base?